src/tests/ftest/scrubber/aggregation.py

```diff
@@ -0,0 +1,72 @@
+"""
+  (C) Copyright 2018-2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import time
+
+from telemetry_utils import TelemetryUtils
+from scrubber_test_base import TestWithScrubber
+
+
+class TestScrubberEvictWithAggregation(TelemetryUtils, TestWithScrubber):
+    # pylint: disable=too-many-ancestors
+    """Inject Checksum Fault with scrubber enabled
+    and scrubber threshold set to a certain value.
+    Aggregation is run on the background.
+
+    :avocado: recursive
+    """
+    def test_target_eviction_during_aggregation(self):
+        """JIRA ID: DAOS-7333
+
+        1. Start the Aggregation task.
+        2. Create checksum faults above scrubber threshold
+        and see whether SSD auto eviction works as expected.
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=hw,large
+        :avocado: tags=scrubber,faults
+        :avocado: tags=TestScrubberEvictWithAggregation,test_target_eviction_during_aggregation
+        """
+        initial_metrics = {}
+        final_metrics = {}
+        self.add_pool()
+        # Disable the aggregation on the pool.
+        self.pool.set_property("reclaim", "disabled")
+        self.add_container(self.pool)
+        # Pool and Containers are already created. Just run the IOR.
+        self.run_ior_with_pool(create_cont=False)
+        initial_aggregation_metrics = self.get_metrics("engine_pool_vos_aggregation_obj_scanned")
+        # Enable the aggregation on the pool.
+        self.pool.set_property("reclaim", "time")
+        # Now enable the scrubber on the pool.
+        self.pool.set_property("scrub", "timed")
+        self.pool.set_property("scrub-freq", "1")
+        self.pool.set_property("scrub-thresh", "3")
```

Comment:
Just FYI there is another function for this if you want to update all at once
```suggestion
        self.pool.set_prop(properties="scrub:timed,scrub-freq:1,scrub-thresh:3")
```

Reply:
done.